packages/runtime/test/src/interpreter/opcode.ts

```diff
@@ -12,26 +12,26 @@ describe("Teal Opcodes basic assertions", function () {
   it("check uint64 overflow", function () {
     const max = MAX_UINT64 + BigInt("5");
     expectTealError(
-      () => op.checkOverflow(max),
+      () => op.checkOverflow(max, 1),
       ERRORS.TEAL.UINT64_OVERFLOW
     );
   });
 
   it("check uint64 underflow", function () {
     const min = MIN_UINT64 - BigInt("1");
     expectTealError(
-      () => op.checkUnderflow(min),
+      () => op.checkUnderflow(min, 1),
       ERRORS.TEAL.UINT64_UNDERFLOW
     );
   });
 
   it("check minimum stack length", function () {
     const stack = new Stack<StackElem>();
-    op.assertMinStackLen(stack, 0);
+    op.assertMinStackLen(stack, 0, 1);
```

Comment:
I think it can be confusing. From an outside eye i am not sure whether i am asserting for stack length `0` or `1`. Maybe we could pass source line in an object (eg. `op.assertMinStackLen(stack, 0, { line: 1})` ) ?

Reply:
i don't follow, because there are many functions similar to this, we cannot pass objects instead of arguments(in every function),
we are in test folder, i think it will be better if we can store it as `line` `stackLength`.
In src we have name for arguments

Reply:
> we are in test folder, i think it will be better if we can store it as line stackLength.

That works too! Could you update for similar tests as well? Otherwise `op.checkOverflow(2, 1)`, `op.assertMinStackLen(stack, 0, 1)`... seems confusing.

Reply:
yes